WORKSPACE

```diff
@@ -183,14 +186,6 @@ http_archive(
     urls = ["https://download.microsoft.com/download/5/F/0/5F0362BD-7D0A-4A9D-9BF9-022C6B15B04D/dotnet-runtime-2.0.0-linux-x64.tar.gz"],
 )
 
-# For the debug image
-http_file(
-    name = "busybox",
-    executable = True,
-    sha256 = "b51b9328eb4e60748912e1c1867954a5cf7e9d5294781cae59ce225ed110523c",
-    urls = ["https://busybox.net/downloads/binaries/1.27.1-i686/busybox"],
-)
-
```

Comment:
Why do we remove the version checks? We seem to check for version 22 below, we should insert here the right sha256 check

Reply:
This PR just gets busybox from theh Debian snapshot repo rather than downloading the binary directory from busybox.net. And Distroless does verify all checksums when downloading Debian packages. Marking resolved.